[READY] add blog about LS workshop

RonaldDijkstra opened the pull request:
Recap blog about the LS workshop in Zwolle by @IwannaDJ

tarzan commented:
Don't we want to position ourselves as THE 'to-go-to-authority' on knowledge building and sharing? I think we could use these blog-posts to position our opinion and vision a bit more strongly. Would really like our audience to think: 'those guys (and girls) at Defacto, they really are onto something!'

fatboypunk commented on source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown:
```diff
@@ -0,0 +1,62 @@
+
+## Fans
+
+Mooie reactie op een product waar mijn collega's en ik zelf al fan van zijn. Bij Defacto gebruiken we het voor het delen van ál onze expertise. [Probeer LearningSpaces de eerste zestig dagen gratis](http://www.learningspaces.nl){:target="_blank"} en wordt ook fan ;).
```
zestig mag je volgens mij als `60` schrijven

dkhgh replied:
regurgitatie. regurgitatie zelfst.naamw. [medisch] terugstroming (meestal van voedsel uit maag naar mond)

RonaldDijkstra replied:
:nauseated_face: 

We should use something else here.

fatboypunk commented on source/blog/nl/2016-12-20-waardevolle-kennis-delen-met-learningspaces.html.markdown:
```diff
@@ -0,0 +1,62 @@
+
+Je kunt ook deelnemen aan één van onze workshops of een demo. Neem daarvoor contact op met [Iwanna de Jonge](mailto:iwanna@defacto.nl) of bel met Defacto op 050 31 44 832.
+
+<!-- De middag begint met een praatje van Marcel Horlings, developer van LearningSpaces, over de visie achter het product. Hij legt uit wat de bestaansreden van LearningSpaces is: *"We hebben veel [zwaardere LMS applicaties gezien](http://blog.learningspaces.io/ending-the-lms-era-for-organisations/){:target="_blank"} en dachten dat dit makkelijker kon. Plus, gaan we nog wel op dezelfde manier met ...
```
So i Guess we can remove this right?

RonaldDijkstra replied:
💩  oops

RonaldDijkstra commented:
- Removed some unnecessarily content 
- Changed publish date to today
- Changed the sentence with regurgitatie a little 

Please approve my changes

RonaldDijkstra commented:
👍
